modules/core/src/main/scala/org/scalasteward/core/forge/github/GitHubApiAlg.scala

```diff
@@ -99,7 +99,7 @@ final class GitHubApiAlg[F[_]](
       F.raiseWhen(repoOut.archived)(RepositoryArchived(repo))
     }
 
-  /** https://docs.github.com/en/rest/pulls?apiVersion=2022-11-28#list-pull-requests */
+  /** https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests */
```

Comment:
This new url is the current one for the API documentation on the endpoint: https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests